department-of-veterans-affairs/next-build
# 14996 dev info in head

## Description
Relates to #[14996](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/14996). (or closes?)


## Testing done
Local vision testing

## Screenshots
![image](https://github.com/department-of-veterans-affairs/next-build/assets/61624970/1e0e3aa8-5d1a-4acb-9191-a38ea5229a5f)


## QA steps
- Navigate to a story listing, story, QA or personal profile page
- Verify that comment appears with relevant info


## Is this PR blocked by another PR?
- Add the `DO NOT MERGE` label
- Add links to additional PRs here:
